### PR TITLE
draft-ietf-secevent-http-push last call feedback

### PR DESCRIPTION
**Pull request by mscurtescu**
Editorial changes based on feedback from these threads:
* https://mailarchive.ietf.org/arch/msg/id-event/mG-Uto9grX_La5lmi5ZR64X8EmI
* https://mailarchive.ietf.org/arch/msg/id-event/EfLHdOri6BiquWE4YKYQqedW9kA
* https://mailarchive.ietf.org/arch/msg/id-event/a7ikHnrCjl8mjdce3nJBuMUGvS8

### CODE REVIEW

File: draft-ietf-secevent-http-push.xml [hunk 1]
```diff
@@ -101,7 +101,7 @@
                     when, and only when, they appear in all capitals, as shown here.
                 </t>
 
-                <t>Throughout this documents all figures may contain spaces and
+                <t>Throughout this document all figures may contain spaces and
```
Comment (selfissued) [inline draft-ietf-secevent-http-push.xml]: Add a comma after "document".

File: draft-ietf-secevent-http-push.xml [hunk 2]
```diff
@@ -172,8 +172,8 @@
             </t>
             <t>
                 SET Transmitters MAY transmit SETs issued by another entity. The SET
-                Recipient may accept or reject (i.e., return an <spanx style="verb">access_denied</spanx>
-                error response) at its own discretion.
+                Recipient may accept or reject (i.e., return an error response such as
+                <spanx style="verb">access_denied</spanx>) at its own discretion.
```
Comment (selfissued) [inline draft-ietf-secevent-http-push.xml]: Add "a SET" before "at its own discretion".  Currently the sentence doesn't say what it is that would be accepted or rejected.

**Review by selfissued — APPROVED** [review]
Please apply my minor suggestions before merging.

File: draft-ietf-secevent-http-push.xml (hunk 1 above)
Comment (mscurtescu) [reply draft-ietf-secevent-http-push.xml]: done

File: draft-ietf-secevent-http-push.xml (hunk 2 above)
Comment (mscurtescu) [reply draft-ietf-secevent-http-push.xml]: done

**Comment by mscurtescu** [comment]
Mike, please merge when you get a chance. I cannot merge since I don't have write access.

**Review by selfissued — APPROVED** [review]
Looks good

**Pull request merged**